packages/Search/test/tstLinearBVH.cpp

```diff
@@ -11,7 +11,7 @@
 
 #include <Teuchos_UnitTestHarness.hpp>
 
-#include <boost/geometry/index/rtree.hpp>
+#include <boost/geometry.hpp>
```

Comment:
did they change the header install structure in 1.56?

Reply:
It looks like they did but `<boost/geometry.hpp>` will work with older versions so I think that's the way to go.